min_dom_set_z3.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import time
+import pygraphviz as pgv
+from z3 import *
+import sys
+
+def min_dom_set(graph):
+    """Try to dominate the graph with the least number of verticies possible"""
+    s = Optimize()
```

Comment:
What do you think about adding a link to http://rise4fun.com/Z3Opt/tutorial/guide? And mentioning this is available from 4.4.1 only?